CondCore/CTPPSPlugins/src/plugin.cc

```diff
@@ -48,5 +52,7 @@ REGISTER_PLUGIN(PPSDirectSimulationDataRcd, PPSDirectSimulationData);
 REGISTER_PLUGIN(PPSPixelTopologyRcd, PPSPixelTopology);
 REGISTER_PLUGIN(PPSAlignmentConfigRcd, PPSAlignmentConfig);
 REGISTER_PLUGIN(PPSAlignmentConfigurationRcd, PPSAlignmentConfiguration);
+REGISTER_PLUGIN(TotemAnalysisMaskRcd, TotemAnalysisMask);
+REGISTER_PLUGIN(TotemReadoutRcd, TotemDAQMapping);
```

Comment:
@tvami should we use `REGISTER_PLUGIN_NO_SERIAL` or `REGISTER_PLUGIN` ?

I am asking as I've seen many changes like that in recent HDF5 PR:
![image](https://github.com/cms-sw/cmssw/assets/7374733/01ab90dc-3e28-4ef6-9bd2-535aea5d9369)

Reply:
Let me tag @Dr15Jones about this

Reply:
If the same class is mapped to multiple records:
  - One registering should use `REGISTER_PLUGIN`
  - All the others should use `REGISTER_PLUGIN_NO_SERIAL`
 
In any case, IIUC the code won't compile if you have implemented it wrongly.
Chris, please correct me if I said something wrong.